projects/NXP/devices/iMX6/bootloader/canupdate.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
Then  [usr/share/bootloader/canupdate.sh](https://github.com/LibreELEC/LibreELEC.tv/blob/master/projects/NXP/devices/iMX6/filesystem/usr/share/bootloader/canupdate.sh) should be dropped?

Reply:
I'm not sure what you're trying to point out?

Reply:
Yes, it can be dropped. I'll add that later.